src/modules/shared/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button } from '@meemoo/react-components';
+import { useTranslation } from 'next-i18next';
+import { FC } from 'react';
+
+import { Modal } from '../Modal';
+
+import { ConfirmationModalProps } from './ConfirmationModal.types';
+
+const ConfirmationModal: FC<ConfirmationModalProps> = ({
+	title,
+	description,
+	onConfirm,
+	onCancel,
+	onClose,
+	isOpen,
+}) => {
+	const [t] = useTranslation();
+
+	const renderButtons = () => {
+		return (
+			<div className="u-text-center u-mb-48">
+				<Button
+					label={t(
+						'modules/shared/components/confirmation-modal/confirmation-modal___ja'
+					)}
+					name={'Ja'}
+					onClick={onConfirm}
+					variants={['black']}
+				/>
+				<Button
+					label={t(
+						'modules/shared/components/confirmation-modal/confirmation-modal___nee'
+					)}
+					className="u-ml-8"
+					name={'Ja'}
```

Comment:
Moet dit `nee` zijn?

Reply:
Fixed in https://github.com/viaacode/hetarchief-client/pull/165/files#diff-3ac0af410a6638a14cd78683b38caec14be25dc97d8aa487bcd12b9178496c81R28